HomeUI/src/views/apps/RegisterFluxApp.vue

```diff
@@ -1761,7 +1761,7 @@ export default {
       return continents;
     },
     countriesOptions(continentCode, isNegative) {
-      const countries = [{ value: isNegative ? 'NONE' : 'ALL', text: isNegative ? 'NONE' : 'ALL' }];
```

Comment:
I think this was ok, on forbidden, if none is selected on continent it should say None not all. As it will not set anything if the it's on the default choice.